download_and_gen.sh

```diff
@@ -192,6 +219,38 @@ do
     > ${PMU_EVENTS_PATH}/${ARCH_NAMES[$short],,}/${short,,}-metrics.json
 done
 
+# Extract hybrid metrics from TMA_Metrics.csv and E-core-TMA_Metrics.csv
+for short in "${!HYBRID_TMA_MODELS_CORE[@]}"
+do
+  core=${HYBRID_TMA_MODELS_CORE[$short]}
+  core_path=${PMU_EVENTS_PATH}/${ARCH_NAMES[$short],,}/${short,,}-core-metrics.json
+  atom=${HYBRID_TMA_MODELS_ATOM[$short]}
+  atom_path=${PMU_EVENTS_PATH}/${ARCH_NAMES[$short],,}/${short,,}-atom-metrics.json
+  echo -e "\\n\\n\\nProcessing hybird core TMA metrics for $core\\n\\n\\n"
+  python3 extract-tma-metrics.py \
+    --memory \
+    --cstate \
```

Comment:
Only one of "cpu_core" and "cpu_atom" can enable "--cstate", if both are enabled, there should be two same "cstate" metrics. I suggest only enabling the "--cstate" for the later PMU "cpu_atom".

Reply:
Thanks, done.